docker/gp_tests/scripts/configs/common_config.json

```diff
@@ -11,4 +11,5 @@
 "WALG_PGP_KEY_PATH": "/tmp/PGP_KEY",
 "WALG_GP_SEG_POLL_INTERVAL": "5s",
 "WALG_GP_SEG_UPD_INTERVAL": "1s",
-"WALG_GP_AOSEG_SIZE_THRESHOLD": "0"
\ No newline at end of file
+"WALG_GP_AOSEG_SIZE_THRESHOLD": "0",
+"WALG_FORCE_DELTA": "true"
```

Comment:
do we really need it in common config? Is it intended?

Reply:
Found that there is no wal-log based delta backup test in gp_tests. Why doesn't gp support this feature as postgres? @x4m

Reply:
Ugh, sorry, I missed notification. This feature was not actively used for PG, so noone stood up to make it work for GP...